x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/cloud/instructions.tsx

```diff
@@ -34,10 +34,16 @@ export const CloudSetupInstructions: React.FC<Props> = ({ productName, cloudDepl
                   values={{
                     editDeploymentLink: cloudDeploymentLink ? (
                       <EuiLink href={cloudDeploymentLink + '/edit'} target="_blank">
-                        edit your deployment
+                        {i18n.translate(
+                          'xpack.enterpriseSearch.setupGuide.cloud.step1.instruction1LinkText',
+                          { defaultMessage: 'edit your deployment' }
+                        )}
                       </EuiLink>
                     ) : (
-                      'Visit the Elastic Cloud console'
+                      i18n.translate(
+                        'xpack.enterpriseSearch.setupGuide.cloud.step1.instruction1LinkText',
+                        { defaultMessage: 'edit your deployment' }
+                      )
```

Comment:
I thought about DRYing this out but was lazy about it, would prefer to address later if possible since there's some time considerations in getting this fix in